google/cloud/storage/benchmarks/storage_shard_throughput_benchmark.cc

```diff
@@ -0,0 +1,481 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/internal/build_info.h"
+#include "google/cloud/internal/format_time_point.h"
+#include "google/cloud/internal/getenv.h"
+#include "google/cloud/internal/random.h"
+#include "google/cloud/storage/benchmarks/benchmark_utils.h"
+#include "google/cloud/storage/client.h"
+#include <deque>
+#include <future>
+#include <iomanip>
+#include <sstream>
+
+namespace {
+namespace gcs = google::cloud::storage;
+namespace gcs_bm = google::cloud::storage_benchmarks;
+
+char const kDescription[] = R"""(
+A throughput benchmark for the Google Cloud Storage C++ client library.
+
+This program first creates a bucket that will contain all the GCS objects used
+in the test.  The bucket is deleted at the end of the test. The name of the
+bucket is selected at random, so multiple instances of this benchmark can run
+simultaneously. The bucket uses the `STANDARD` storage class, in a region set
+via the command-line.
+
+After creating this bucket the program creates a number of objects, all the
+objects have the same contents, but the contents are generated at random.
+
+The size of the objects can be configured in the command-line, but they are
+typically 250MiB is size. The objects are constructed by writing N identical
```

Comment:
s/is size/in size/

And perhaps "typically" should be "by default"?

Reply:
Fixed, thanks.